website/docs/reference/warehouse-profiles/spark-profile.md

```diff
@@ -31,17 +39,24 @@ your_profile_name:
   outputs:
     dev:
       type: spark
-      method: thrift
+      method: odbc
+      driver: [path/to/driver]
       schema: [database/schema name]
-      host: [hostname]
-      port: [port]
-      user: [user]
+      host: [yourorg.sparkhost.com]
+      organization: [org id]    # Azure Databricks only
+      port: [port]              # default 443
```

Comment:
Confused about this comment — does this mean:
- If you don't include it, dbt will assume it is 443 OR
- For most connections, the value will be 443